api/db-cdr/changelog-schema/db.changelog-5.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="srushtigangireddy" id="changelog-5">
+    <validCheckSum>ANY</validCheckSum>
+    <sql dbms="mysql">
+      ALTER TABLE db_domain
```

Comment:
@danqrodney , @freemabd1  This is how we are altering schema now correct ?  Do we want the valid checksum to be any still ?  Liquibase docs say this about validchecksum : 

validCheckSum | List checksums which are considered valid for this changeSet, regardless of what is stored in the database. Used primarily when you need to change a changeSet and don't want errors thrown on databases on which it has already run (not a recommended procedure). Since 1.7
-- | --

As i understand it , when you have validchecksum , you are not allowed to change the changeSet anymore or it will throw an error. Which sucks for development but is good for pushing to environments . I think that since this is a brand new file, we can remove validchecksumany and probably should start having that removed for anything going into master. At least for the schema changes.

Reply:
yes

Reply:
Removed validchecksum any.